src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberContainerSymbol_ImplementationChecks.cs

```diff
@@ -836,7 +836,7 @@ private void CheckNewModifier(Symbol symbol, bool isNew, DiagnosticBag diagnosti
                             {
                                 var overridenParameterType = overriddenParameters[i].Type;
                                 if (!overridingParameters[i].Type.Equals(overridenParameterType,
-                                                                         TypeCompareKind.AllIgnoreOptions & ~TypeCompareKind.IgnoreNullableModifiersForReferenceTypes) &&
+                                                                         TypeCompareKind.AllIgnoreOptions & ~(TypeCompareKind.IgnoreNullableModifiersForReferenceTypes | TypeCompareKind.IgnoreInsignificantNullableModifiersDifference)) &&
```

Comment:
>TypeCompareKind.AllIgnoreOptions & ~(TypeCompareKind.IgnoreNullableModifiersForReferenceTypes | TypeCompareKind.IgnoreInsignificantNullableModifiersDifference) [](start = 73, length = 159)

Consider adding a `TypeCompareKind.AllNonNullableIgnoreOptions`.

Reply:
This and other similar places are still using UnknownNullableModifierMatchesAny, which is one of the nullable ignore options.

---
In reply to: [232119400](https://github.com/dotnet/roslyn/pull/30913#discussion_r232119400) [](ancestors = 232119400)